NEW_README.md

```diff
@@ -75,3 +76,17 @@ From root of the gatling-puppet-load-test source dir on the Gatling driver (metr
     * scp them locally and you can view them in your browser.
 * atop files containing cpu, mem, disk and network usage overall and broken down per process are available to view on the mom: atop -r /var/log/atop/atop_\<date>
     * See the atop man file for interactive commands
+    
+### Aborting a test run
+The easiest way to immediately kill a test run is to kill the corresponding Java process on the metrics node. Find the process id with top and then use `kill <PID>`.
+
+### Generating reports for aborted runs
+If a Gatling scenario is aborted the reports may not be generated. In this case you can run Gatling in reports-only mode to generate the reports for a previous run. 
+You'll need to specify the name of the folder on the metrics node containing the simulation.log file for the run. 
+Look in `~/gatling-puppet-load-test/simulation-runner/results` for a folder that starts with 'PerfTestLarge-' followed by the run id and verify that the folder contains a valid simulation.log file.
+
+If you find that the simulation.log file is not being populated during your run you may need to reduce the log buffer size from the 8kb default.
+Set `gatling.data.file.bufferSize` in gatling.conf to a smaller value like 256 (this may impact performance).
```

Comment:
Can you document where gatling.conf is here.

Reply:
Will do.